dali/operators/signal/fft/spectrogram.cc

```diff
@@ -27,39 +27,58 @@
 namespace dali {
 
 DALI_SCHEMA(Spectrogram)
-  .DocStr(R"code(Produces a spectrogram from a 1D signal (e.g. audio). Input data is expected
-to be single channel (shape being `(nsamples,)`, `(nsamples, 1)` or `(1, nsamples)`) of type
-float32.)code")
+  .DocStr(R"code(Produces a spectrogram from a 1D signal (for example, audio).
+
+Input data is expected to be one channel (shape being ``(nsamples,)``, ``(nsamples, 1)``, or
+``(1, nsamples)``) of type float32.)code")
   .NumInput(1)
   .NumOutput(1)
   .AddOptionalArg("nfft",
-    R"code(Size of the FFT. The number of bins created in the output is `nfft // 2 + 1`
-(positive part of the spectrum only).)code",
+    R"code(Size of the FFT.
+
+The number of bins that are created in the output is ``nfft // 2 + 1``.
+
+.. note::
+  The output only represents the positive part of the spectrum.)code",
     -1)
   .AddOptionalArg("window_length",
-    R"code(Window size (in number of samples))code",
+    R"code(Window size in number of samples.)code",
     512)
   .AddOptionalArg("window_step",
-    R"code(Step betweeen the STFT windows (in number of samples))code",
+    R"code(Step betweeen the STFT windows in number of samples.)code",
     256)
   .AddOptionalArg("window_fn",
     R"code(Samples of the window function that will be multiplied to each extracted window when
-calculating the STFT. If provided it should be a list of floating point numbers of size
-`window_length`. If not provided, a Hann window will be used.)code",
+calculating the STFT.
+
+If a value is provided, it should be a list of floating point numbers of size ``window_length``.
+If a value is not provided, a Hann window will be used.)code",
     std::vector<float>{})
   .AddOptionalArg("power",
-    R"code(Exponent of the magnitude of the spectrum. Supported values are 1 for energy and 2 for
-power.)code",
+    R"code(Exponent of the magnitude of the spectrum.
+
+Here are the supported values:
+
+- ``1`` for energy.
+- ``2`` for power.
+)code",
     2)
   .AddOptionalArg("center_windows",
-    R"code(Indicates whether extracted windows should be padded so that window function is centered
-at multiples of `window_step`. If set to false, the signal will not be padded, that is only windows
-within the input range will be extracted.)code",
+    R"code(Indicates whether extracted windows should be padded so that the window function is
+centered in multiples of ``window_step``.
```

Comment:
```suggestion
centered at multiples of ``window_step``.
```
We meant exactly that.

Reply:
Done